src/setup-wp-env/create-wp-env-config.ts

```diff
@@ -0,0 +1,91 @@
+import { minimist, fs } from 'zx';
+
+async function main() {
+	const {
+		wp,
+		php,
+		plugins,
+		themes,
+		mappings,
+		dir,
+		'active-theme': activeTheme,
+	} = getOptions({
+		wp: { type: 'string', default: null },
+		php: { type: 'string', default: null },
+		plugins: { type: 'string', default: '' },
+		themes: { type: 'string', default: '' },
+		'active-theme': { type: 'string', default: '' },
+		mappings: { type: 'string', default: '' },
+		dir: { type: 'string', default: './' },
```

Comment:
maybe rename dir to something more informative?

Reply:
Done